org.openkj.OpenKJ.yml

```diff
@@ -0,0 +1,173 @@
+        tag: 0.14.0
+        commit: 73284b676b12b47e17af2ef1b430527299e10c17
+      - type: script
+        commands:
+          - "autoreconf -fiv"
+        dest-filename: autogen.sh
+  - name: ffmpeg
+    cleanup:
+      - "/lib/codecs/share/ffmpeg/examples"
+    config-opts:
+      - "--disable-debug"
+      - "--disable-doc"
+      - "--disable-static"
+      - "--enable-shared"
+      - "--enable-nonfree"
+      - "--enable-gpl"
+    sources:
+      - type: git
+        url: "https://git.ffmpeg.org/ffmpeg.git"
+        tag: n4.2.4
+        commit: f9f95ceebfbd7b7f43c1b7ad34e25d366e6e2d2b
+  - name: gst-libav
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=disabled"
+    sources:
+      - type: git
+        url: "https://gitlab.freedesktop.org/gstreamer/gst-libav.git"
+        tag: 1.16.3
+        commit: 6988bdb0482b18b7a9f0c9635217a93d5057a9ff
+      - type: patch
+        path: patches/gst-libav-stop-caching-codecs.patch
+    post-install:
+      - "mkdir -p /app/lib/codecs/lib/gstreamer-1.0"
+      - "mv /app/lib/gstreamer-1.0/*.so /app/lib/codecs/lib/gstreamer-1.0"
+  - name: gst-plugins-bad
+    buildsystem: meson
+    cleanup:
+      - "/bin/*webrtc*"
+      - "/bin/crossfade"
+      - "/bin/tsparser"
+      - "/bin/playout"
+    config-opts:
+      - "-Dopenh264=disabled"
+      - "-Dvdpau=disabled"
+      - "-Dvulkan=disabled"
+    sources:
+      - type: git
+        url: "https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad.git"
+        tag: 1.16.3
+        commit: ee8144e98b084d75ffabaef0ef3dca2af8d72061
+    post-install:
+      - "mkdir -p /app/lib/codecs/lib/gstreamer-1.0"
+      - "mv /app/lib/gstreamer-1.0/*.so /app/lib/codecs/lib/gstreamer-1.0"
+  - name: rubberband 
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/breakfastquay/rubberband.git
+        commit: f1acaf85d70dac991c835626fe768fc0a8d3cb52
+  - name: fmt 
+    buildsystem: cmake-ninja
+    builddir: yes
+    config-opts:
+      - "-DFMT_TEST=OFF"
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt.git
+        commit: 561834650aa77ba37b15f7e5c9d5726be5127df9
+  - name: spdlog 
+    buildsystem: cmake-ninja
+    builddir: yes
+    config-opts:
+      - "-DSPDLOG_BUILD_EXAMPLE=OFF"
+      - "-DSPDLOG_BUILD_TESTS=OFF"
+    sources:
+      - type: git
+        url: https://github.com/gabime/spdlog.git
+        commit: bee3e63e1bae8b4cd5e3ebd69fbf75f68e47996a
+  - name: taglib 
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/taglib/taglib.git
+        commit: f58161511050ac6c060d28e75dcd5c5a3af7279c 
+  - name: OpenKJ
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/openkj/openkj.git
+        tag: v2.0.4-release
+        commit: dd102391f4cd57728f215ef6e26b37794e569ffb 
+cleanup-commands:
+  - "mv /app/lib/liba52*.* /app/lib/codecs/lib/"
+  - "mv /app/lib/libass.* /app/lib/codecs/lib/"
```

Comment:
I'd move it to liba52/libass respectively, but it's not a hill I'm willing to die on.

Reply:
(Feel free to handle it after the merge.)